src/payment/strategies/mollie/mollie-script-loader.ts

```diff
@@ -0,0 +1,25 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { MollieClient, MollieHostWindow } from './mollie';
+
+export default class MollieScriptLoader {
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        private _window: MollieHostWindow = window
+    ) {}
+
+    async load(merchantId: string, locale: string, testmode: boolean): Promise<MollieClient> {
+        await this._scriptLoader.loadScript('https://js.mollie.com/v1/mollie.js');
+
+        if (!this._window.Mollie) {
```

Comment:
ideally the best way to check for this is using a typeguard as suggested before.

This is an example

https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/analytics/is-analytics-step-tracker-window.ts
https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/analytics/analytics-tracker-window.ts

Notice the AnalyticsTracker is not an optional attribute of the `AnalyticsTrackerWindow` interface, which is what I suggested for the `MolllieWindow` interface